app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="well panel panel-default">
```

Comment:
When it comes down to `Profile` I don't see a point in a `show` page, unless it is used for display for other people, i.e. public profile. Considering that I'm not sure if you want some of the information on this page to stay on it.

I would direct the `current_user` directly to the edit page.